packages/augur-ui/src/modules/app/components/app.tsx

```diff
@@ -99,16 +100,18 @@ export default class AppView extends Component<AppProps> {
     {
       title: 'Disputing',
       route: DISPUTING,
+      requireLogin: false,
     },
     {
       title: 'Reporting',
       route: REPORTING,
+      requireLogin: false,
     },
     {
       title: 'Create',
       route: CREATE_MARKET,
       requireLogin: true,
-      disabled: this.props.universe.forkingInfo,
+      disabled: this.props.universe.forkingInfo !== null,
```

Comment:
this needs to be !!this.props.universe.forkingInfo

it's possible to be undefined